src/api/docker-files/Dockerfile

```diff
@@ -22,6 +22,9 @@ ADD Gemfile /obs/src/api/Gemfile
 ADD Gemfile.lock /obs/src/api/Gemfile.lock
 RUN chown -R frontend /obs/src/api
 
+# Remove chromedriver binary to allow selenium-manager manage versions
+RUN rm -rf /usr/bin/chromedriver
```

Comment:
This is a temporary hack. The proper way to do it is to remove chromedriver from the docker image we use in the CI pipeline, or just use [frontend-base](https://build.opensuse.org/package/show/OBS:Server:Unstable:Containers/frontend-base) (that does not have chromedriver installed) instead of [frontend-features](https://build.opensuse.org/package/show/OBS:Server:Unstable:Containers/frontend-features) as docker image

Reply:
please just make sure that this temporary hack is recorded in a card, so we can follow up and do it properly.

Reply:
> please just make sure that this temporary hack is recorded in a card, so we can follow up and do it properly.

https://trello.com/c/IYwr5IsM/194-remove-frontend-features-container

Reply:
As I said before elsewhere, this does not change anything in CI...

Reply:
Why not?